python/doc/source/development.rst

```diff
@@ -76,13 +76,23 @@ Using Conda
 Let's create a conda environment with all the C++ build and Python dependencies
 from conda-forge:
 
+On Linux and OSX:
+
+.. code-block:: shell
+
+    conda create -y -n pyarrow-dev -c conda-forge \
+        --file arrow/ci/conda_env_cpp.yml \
+        --file arrow/ci/conda_env_python.yml \
+        python=3.6
+
+On Windows:
+
 .. code-block:: shell
 
-   conda create -y -q -n pyarrow-dev \
-         python=3.6 numpy six setuptools cython pandas pytest \
-         cmake flatbuffers rapidjson boost-cpp thrift-cpp snappy zlib \
-         gflags brotli jemalloc lz4-c zstd -c conda-forge
-   conda activate pyarrow-dev
+    conda create -y -n pyarrow-dev -c conda-forge ^
+        --file arrow\\ci\\conda_env_cpp.yml ^
+        --file arrow\\ci\\conda_env_python.yml ^
```

Comment:
Hm, I think just single backslashes is OK? I will try it out on Windows before merging

Reply:
I'll render the docs, because I put those to escape the single backslash.

Reply:
Fixed it